governing-documents/MAINTAINERS-guidelines.md

```diff
@@ -28,17 +28,17 @@ In most repositories, the "Maintainer" scope is given to all repository
 Maintainers. The "Maintainer" scope is defined as the "Maintain" GitHub role.
 This is usually sufficient to do all of the work required of a Maintainer. From
 time to time, Maintainers may have to request certain administrative tasks by
-performed by the Hyperledger GitHub organization administrators.
+performed by the LFDT GitHub organization administrators.
 
 In some repositories, the maintainers may decide to define additional scopes and
 each Maintainer given one or more of those designated scopes:
 
-- In rare cases, Maintainers in a repository may request that the Hyperledger Foundation GitHub organization administrators enable designating some Maintainers with different GitHub roles, such as "Admin" (more capable than "Maintain") or "Triage" (less capable than "Maintain").
-  - Each elevated GitHub role needed in a repository requires the Hyperledger Foundation GitHub organization administrators to create and maintain an additional GitHub Team.
-  - The Hyperledger Foundation GitHub organization administrators manage a team per Hyperledger Project that includes all contributors to the project. That team is given the "Read" GitHub role in all project repositories. This team need not be documented in the "MAINTAINERS" file.
+- In rare cases, Maintainers in a repository may request that the LFDT TAC enable designating some Maintainers with different GitHub roles, such as "Admin" (more capable than "Maintain") or "Triage" (less capable than "Maintain").
+  - Each elevated GitHub role needed in a repository requires the LFDT GitHub organization administrators to create and maintain an additional GitHub Team.
```

Comment:
I forgot to update this, but I'll do so after my above question is answered.

Reply:
@VRamakrishna I put down my thoughts, these are just a vote though. No strong opinions on either of those answers that I provided and if someone else from the TOC feels strongly in the opposite direction I'm happy to go along.

Reply:
I think this ought to be the project's TSC too.